files/build_templates/docker_image_ctl.j2

```diff
@@ -60,6 +60,17 @@ function getMountPoint()
     echo $1 | python -c "import sys, json, os; mnts = [x for x in json.load(sys.stdin)[0]['Mounts'] if x['Destination'] == '/usr/share/sonic/hwsku']; print '' if len(mnts) == 0 else os.path.abspath(mnts[0]['Source'])" 2>/dev/null
 }
 
+function isPlatformReset()
+{
+    output=$(python3 -c "import sonic_platform.platform; p = sonic_platform.platform.Platform(); c = p.get_chassis(); hw_rc_major, hw_rc_minor = c.get_reboot_cause(); print(hw_rc_major)" 2>/dev/null)
+    echo "${output}"
```

Comment:
Maybe this should echo more context, rather than just the reboot cause?

Reply:
I commented about it yesterday but it said pending status.